keyboards/retne/bonsai/readme.md

```diff
@@ -0,0 +1,15 @@
+# Bonsai
+
+![Bonsai](https://i.imgur.com/XypHguN.jpg)
+
+A 75% take on the Alice that can be assembled with only through hole components, including usb type-c
+
+* Keyboard Maintainer: [Retne](https://github.com/Retne01)
+* Hardware Supported: Bonsai, atmega32a
+* Hardware Availability: [GitHub](https://github.com/Retne01/Bonsai)
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add information how to jump to the bootloader?

Reply:
I've got a guide put together on the github for this board.  It covers how to acquire the parts, solder the board together, how to flash the bootloader to the MCU, and finally how to flash QMK to the MCU.  Would it be worth linking to it here?  If not should I include how to do it after QMK has been loaded onto the keyboard or before?  I included the reset command on Fn+R but the board can also be reset by holding down the boot button and tapping the reset button.

Reply:
Ideally, a short bit about how to jump to bootloader should be included in the readme here.

Or at least, a link to the repo with the docs.

Reply:
Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```